src/main/telemetry/crsf.c

```diff
@@ -105,6 +107,14 @@ typedef struct {
 
 static crsfSpeedControl_s crsfSpeed = {0};
 
+typedef enum {
+    CRSF_LINK_UNKNOWN,
+    CRSF_LINK_ELRS,
+    CRSF_LINK_NOT_ELRS
```

Comment:
CRSF_LINK_ORIGINAL or CRSF_LINK_TBS maybe?

Reply:
I’m looking for ELRS, and anything else, now or in the future, isn’t.